runelite-client/src/main/java/net/runelite/client/plugins/puzzlesolver/PuzzleSolverOverlay.java

```diff
@@ -79,6 +80,7 @@
 	private PuzzleSolver solver;
 	private Future<?> solverFuture;
 	private int[] cachedItems;
+	private boolean useNormalSolver;
```

Comment:
This variable should be a local variable in render(..) since it's overwritten everytime that's run, and it's only in there it is set.
Pass the boolean into ``convertToSolverFormat`` and ``solve`` and you'll be okay.